src/Tribe/View_Helpers.php

```diff
@@ -371,6 +371,8 @@ public static function loadStates() {
 				'WI' => esc_html__( 'Wisconsin', 'tribe-common' ),
 				'WY' => esc_html__( 'Wyoming', 'tribe-common' ),
 			);
+
+			return apply_filters( 'tribe_get_state_options', $states );
```

Comment:
Could you add a docblock for this?

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters